pipeline-steps/artifact_build.groovy

```diff
@@ -1,71 +1,106 @@
-def python(Map args) {
+def get_rpc_repo_creds(){
```

Comment:
I don't think this will work as is.  I see two options:

1. Rename to with_rpc_repo_creds, accept a body closure and execute that within the withCredentials block (see [runonpubcloud](https://github.com/rcbops/rpc-gating/blob/master/pipeline-steps/pubcloud.groovy#L140))
1. Return the array of arguments to be used in withCredentials by the caller.